scene/gui/button.cpp

```diff
@@ -470,6 +496,19 @@ String Button::get_text() const {
 	return text;
 }
 
+void Button::set_autowrap_mode(TextServer::AutowrapMode p_mode) {
+	if (autowrap_mode != p_mode) {
+		autowrap_mode = p_mode;
+		_shape();
+		queue_redraw();
+		update_minimum_size();
+	}
```

Comment:
```suggestion
	if (autowrap_mode == p_mode) {
		return;
	}
	autowrap_mode = p_mode;
	_shape();
	queue_redraw();
	update_minimum_size();
```
Use early exit

Reply:
The rest of the setters in the `LineEdit` use this style, so I have used it as well for consistency.
This probably should be changed everywhere in the separate PR, since different parts of code use both styles.

Reply:
I'd say that the early exit style is the preferred even with consistency, but I'll leave it to you 🙂